test/chpldoc/functions/functionTypeexprFormat.doc.chpl

```diff
@@ -0,0 +1,6 @@
+type t = 2*int;
```

Comment:
I'd name this "typeLooksLikeFunc" instead, I think it would be a little more clear to other people.  I also would file this under `test/chpldoc/globals` instead, since these are variable declarations rather than functions

Reply:
Ok, I shall make the changes.
I am still working on this issue and will let you know as I make some progress.